geoserver/workspaces/OSM/postgis/datastore.xml

```diff
@@ -12,23 +12,23 @@
     <entry key="Max open prepared statements">50</entry>
     <entry key="encode functions">false</entry>
     <entry key="Batch insert size">1</entry>
-    <entry key="preparedStatements">true</entry>
+    <entry key="preparedStatements">false</entry>
     <entry key="database">gis</entry>
     <entry key="host">db</entry>
     <entry key="Loose bbox">true</entry>
     <entry key="Estimated extends">true</entry>
     <entry key="fetch size">1000</entry>
     <entry key="Expose primary keys">false</entry>
     <entry key="validate connections">true</entry>
-    <entry key="Support on the fly geometry simplification">true</entry>
+    <entry key="Support on the fly geometry simplification">false</entry>
     <entry key="Connection timeout">20</entry>
     <entry key="create database">false</entry>
     <entry key="port">5432</entry>
-    <entry key="passwd">crypt1:MXKQfoslmjhNru0yvRSR8Q==</entry>
-    <entry key="min connections">1</entry>
+    <entry key="passwd">crypt1:DPL2XKH4/FXjSVizscF6nA==</entry>
```

Comment:
Not sure why the encrypted password changed, but I did not touch the database and it still works... oh well, at worst you'll have to back out this change.